src/main/java/mate/academy/lib/Injector.java

```diff
@@ -13,8 +13,8 @@
 
 public class Injector {
     private static final Map<String, Injector> injectors = new HashMap<>();
-    private final Map<Class<?>, Object> instanceOfClasses = new HashMap<>();
-    private final List<Class<?>> classes = new ArrayList<>();
+    private static final Map<Class<?>, Object> instanceOfClasses = new HashMap<>();
```

Comment:
It's not a good practice to have static mutable collections, as it can lead to unpredictable behavior in multithreaded environments. Consider refactoring this.